views/gym-routes.html

```diff
@@ -414,10 +415,15 @@ <h1>
                 <a href="/gyms/{{gym['id']}}/edit" alt="{{gym['homepage']}}"><i class="icon-wrench"></i></a>
```

Comment:
Can you add the route name to the screen as well (so that we know which version of the routes we are seeing)?

Reply:
The routes seen are the default routes set in the admin screen but I will see if I can add the name somewhere

Reply:
I added the name to the bottom of the table